src/main/java/SearchingMethods.java

```diff
@@ -1,19 +1,23 @@
+import java.util.Objects;
+
 public class SearchingMethods {
```

Comment:
These are functions :upside_down_face:

Reply:
I thought about changing the name of the class, but seeing another brain with the suggestion, I will make the change.